packages/babel-plugin/__tests__/stylex-transform-stylex-attrs-test.js

```diff
@@ -835,17 +835,11 @@ describe('@stylexjs/babel-plugin', () => {
           import stylex from 'stylex';
           _inject2(".color-x1e2nbdu{color:red}", 3000);
           ({
-            class: "FooBar__styles.default color-x1e2nbdu"
+            class: "color-x1e2nbdu",
+            "data-style-src": "js/FooBar.react.js:3"
```

Comment:
At Meta, I don't think it will add value to display any part of the path before the component name since every file name is unique. In codebases that go the other way, and make heavy use generic component names within folders (e.g., `../Menu/Button/index.js`), the display path might not be long enough to disambiguate from others. So I wonder if it might be necessary to include some kind of configuration for this.

Reply:
I've changed this so that haste modules only display the file name, commonjs displays a shortened path.